parquet-column/src/test/java/org/apache/parquet/column/statistics/TestStatistics.java

```diff
@@ -74,6 +74,13 @@ public void testIntMinMax() {
     assertEquals(statsNeg.getMax(), 54);
     assertEquals(statsNeg.getMin(), -66);
 
+    assertEquals(-1, statsNeg.compareToMax(55));
```

Comment:
These assertions are too specific. The return value can be any positive or negative if the values aren't equal. I think this should use `assertTrue(statsNeg.compareToMax(55) < 0)`

Reply:
Good point. No idea, why I wrote this way...